src/api/app/assets/stylesheets/webui2/groups.scss

```diff
@@ -1,5 +1,27 @@
 #group-users {
   .group-user {
-    min-width: 26rem;
+    width: 19.7rem;
+
+    .group-member-removal {
+      position: absolute;
+      top: 0;
+      right: 0.5rem;
+    }
+  }
+}
+
+@include media-breakpoint-up(xl) {
+  #group-users {
```

Comment:
`#group-users` doesn't need to be repeated in the media queries.

It can be like this:
```
#group-users {
  .group-user {
    width: 19.7rem;
  }

  @include media-breakpoint-up(xl) {
    width: 21.7rem;
  }

  @include media-breakpoint-up(xxl) {
    width: 22.4rem;
  }
}

```